libraries/c_sdk/standard/mqtt/test/system/iot_tests_mqtt_system.c

```diff
@@ -211,6 +211,46 @@ static bool _disconnectSerializerOverride = false;  /**< @brief Tracks if #_disc
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if retries are enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                    const IotMqttConnectInfo_t * pConnectInfo,
+                                    uint32_t timeoutMs,
+                                    IotMqttConnection_t * const pMqttConnection )
+{
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+
+    #if ( IOT_TEST_MQTT_CONNECT_RETRY_ENABLED == 1 )
```

Comment:
Let's make this configurable as the maximum number of retries (instead of just 0/1).

```c
// Set this to the number of connection attempts for the MQTT tests.
// For example, 3 retries.  To disable retries, set to 1.
#define IOT_TEST_MQTT_RETRY_COUNT   3
```

Reply:
ok